test/unit/tracking.test.ts

```diff
@@ -61,22 +49,23 @@ describe("connection user agent", () => {
   it("propagation", async () => {
     const connectionParams: ConnectionOptions = {
       auth: {
-        username: "dummy",
-        password: "dummy"
+        client_id: "dummy",
+        client_secret: "dummy"
       },
-      database: "dummy",
-      engineName: "dummy",
-      account: "account"
+      account: "my_account"
     };
     const firebolt = Firebolt({
       apiEndpoint
     });
 
     server.use(
-      rest.post("https://some_engine.com", (req, res, ctx) => {
-        expect(req.headers.get("user-agent")).toContain("NodeSDK");
-        return res(ctx.status(200), ctx.json({ data: [] }));
-      })
+      rest.post(
+        "https://some_system_engine.com/dynamic/query",
```

Comment:
lets have constant `QUERY_URL` here so we can change it in one place. 
`/dynamic/query` will eventually be `/query`

Reply:
Good point, added.